src/number_theory/arithmetic_function.lean

```diff
@@ -738,6 +780,20 @@ begin
     refl }
 end
 
+lemma list.repeat_dedup {α : Type*} [decidable_eq α] {x : α} :
```

Comment:
This should go in another file, about lists.

Reply:
Oops I thought I'd already moved it. Thanks!!